File: airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/NoOpDestinationV1V2Migrator.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.base.destination.typing_deduping;

public class NoOpDestinationV1V2Migrator implements DestinationV1V2Migrator {

  @Override
  public void migrateIfNecessary(final SqlGenerator sqlGenerator,
                                 final DestinationHandler<?> destinationHandler,
                                 final StreamConfig streamConfig)
      throws TableNotMigratedException, UnexpectedSchemaException {
    // Do nothing
  }

}

File: airbyte-cdk/java/airbyte-cdk/typing-deduping/bin/main/io/airbyte/integrations/base/destination/typing_deduping/NoOpDestinationV1V2Migrator.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.integrations.base.destination.typing_deduping

class NoOpDestinationV1V2Migrator : DestinationV1V2Migrator {
    @Throws(TableNotMigratedException::class, UnexpectedSchemaException::class)
    override fun migrateIfNecessary(
        sqlGenerator: SqlGenerator,
        destinationHandler: DestinationHandler<*>,
        streamConfig: StreamConfig
    ) {
        // Do nothing
    }
}
